.gitignore

```diff
@@ -1,6 +1,10 @@
 *.pyc
```

Comment:
Sorry, but it is inevitable. In order to ignore certain file extensions, you have to add `.gitignore` in your index, hence, it will appear on GitHub.com. Check out https://github.com/cyclus/cyclus as an example